eparticlectr/eparticlectr.cpp

```diff
@@ -357,76 +255,27 @@ void eparticlectr::finalizeextr( uint64_t proposal_id ) {
                 });
             }
         }
-        // Reduce staking time for vote winners to 5 days
-        else if (approved && !vote_it->is_editor) {
+        // Reduce staking time for vote winners to 5 days. Keep losers the same
+        else if (!vote_it->is_editor) {
             auto stake_it = staketable.find(vote_it->stake_id);
-            if (stake_it != staketable.end()){
-                staketable.modify( stake_it, same_payer, [&]( auto& a ) {
-                    a.completion_time = a.timestamp + WINNING_VOTE_STAKE_TIME;
-                });
-            }
-        }
 
-        
-        // Slash losers
-        if (vote_it->approve != approved) {
-            auto stake_it = staketable.find(vote_it->stake_id);
             if (stake_it != staketable.end()){
-                staketable.modify( stake_it, same_payer, [&]( auto& a ) {
-                    a.completion_time += extraSecsSlash;
-                });
-            }
+                if (vote_it->approve == approved){
+                    // Winners
+                    staketable.modify( stake_it, same_payer, [&]( auto& a ) {
+                        a.completion_time = a.timestamp + WINNING_VOTE_STAKE_TIME;
+                    });
+                }
+                else {
+                    // Losers
```

Comment:
should here get the extra slashed time? extraSecsSlash?

Reply:
I think we decided to stop slashing